packages/core/components/Modal/TipsAndTricks/index.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+
+import BaseModal from "../BaseModal";
+import { ModalProps } from "..";
+
+/**
+ * Dialog meant to explain some tips and tricks
+ */
+export default function TipsAndTricks({ onDismiss }: ModalProps) {
+    const body = (
+        <>
+            <h4>Performance</h4>
+            <p>
+                Some searches can be slow. This tends to occur because of the amount of files which
+                can almost always be narrowed down. For example, 5 million files might have
+                annotation &quot;Kind&quot; with a value of &quot;CZI Image&quot;, instead of
+                searching by File name through all those files narrow them down with another
+                annotation or just by something simple like the date they were uploaded. Use the
+                &quot;View&quot; feature at the top of the app to automatically narrow files down by
+                the date they were uploaded relative to todays date.
+            </p>
+            <h4>How to</h4>
+            <p>
+                Unsure how a feature works? Check out the &quot;Tutorials&quot; section of the help
+                menu, a feature might have some hidden aspects you are not aware of or there might
+                even be features there you did not know about.
```

Comment:
>  Unsure how a feature works? Check out the &quot;Tutorials&quot; section of the help
                menu. A feature might have some hidden aspects you are not aware of, or there might
                even be features there you did not know about.